doc/development/seed.md

```diff
@@ -0,0 +1,15 @@
+---
+author: M. Serhat Dundar
+---
+
+# Seed
```

Comment:
İlk bakışta Git LFS için dokümanda bir pasaj göremedim (bu dosyada olması gerekmiyor, diğer dokümanlarda not düşüp belki burada kısaca geçilir).

Reply:
`git-lfs` dokümanının yazılması işini başka müsait bir zamanda yapacağım